docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1446,8 +1446,8 @@ class Item {
 [[resteasy_integration]]
 === RESTEasy Integration
 
-If you want to use Qute in your JAX-RS application, you'll need to add the `quarkus-resteasy-qute` extension first.
-In your `pom.xml` file, add:
+If you want to use Qute in your JAX-RS application, then depending on which JAX-RS stack you are using, you'll need to register the proper extension first.
+If you are using RESTEasy Classic via the `quarkus-resteasy` extension, then in your `pom.xml` file, add:
```

Comment:
```suggestion
If you are using the traditional `quakus-resteasy` extension, then in your `pom.xml` file, add:
```
maybe?

I'm not sure to be honest. I'm just wondering if marketing it as `RESTEasy Classic` vs `RESTEasy Reactive` is a good thing.

Reply:
I don't know either. I've been using that terminology myself but that certainly doesn't mean it's the best

Reply:
But let's go with your suggestion for now, as it doesn't commit us to anything whereas mine pretty much does

Reply:
"traditional `quakus-resteasy`" sounds OK.

Reply:
OK, then the PR now is should be good :)